tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -1413,6 +1413,72 @@ def create_full_val_real_complex128(val):
     assert(isclose(     f_real_complex128(val_float)       ,      create_full_val_real_complex128(val_float), rtol=RTOL, atol=ATOL))
     assert matching_types(f_real_complex128(val_float), create_full_val_real_complex128(val_float))
 
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("python", marks = [
+            pytest.mark.skip,
```

Comment:
Please add a reason to your skip mark. See [here](https://github.com/pyccel/pyccel/blob/master/tests/epyccel/recognised_functions/test_numpy_funcs.py#L3528-L3537) for an example